### DISCUSSION TITLE
OpenWRT - unable to assign LAN to second linux bridge.

### DISCUSSION THREAD

**Discussion by mon5termatt** [tteck/Proxmox#1961] [Help Needed]
### Please verify that you have read and understood the guidelines.

Yes

### A clear and concise description of the issue.

Please reply if you need any more info. i know im not providing a TON of info.

Trying to assign WAN to vmbr0
Trying to assign LAN to vmbr2

Script errors out, see screenshot.

### Which Linux distribution are you employing?

Default Settings

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

![image](https://github.com/tteck/Proxmox/assets/43628254/e27b53a1-ded0-453d-88a7-3416d480a231)

### Please provide detailed steps to reproduce the issue.

Run openWRT script. 

Choose advanced settings and change the LAN bridge to any newly created linux bridge (vmbr2 in my case)

Use defaults for all other options. (I had VLAN changed to 1)

**Comment by mon5termatt** [comment]
Im not even sure if this is possible to do, but I'm trying to make this work like VYOS does. But with a VPN and Web UI

I use vyos to make a private network for sharing a single IP

**Comment by tteck** [comment]
I'm unable to determine if the issue is related to the script or if it's even possible given my lack of experience with it. Until it's determined to be a script issue, it will be moved to discussions.

**Reply by thomasbra1** [reply]
I did get this working my self as I was wanting to test VMs in a sandbox.

To test I created a blank Linux bridge under networking before running the script.  I called it vmbr3

I edited the script to not destroy the VM so I could look at what the VM setting were. 

When do an advanced install I set the LAN to this new bridge.  At the end this failed as the script does not let you avoid adding a VLAN tag to the to the network adapter in the VM.

I could remove the VLAN tag and start the VM properly

![Selection_226](https://github.com/tteck/Proxmox/assets/25257905/998653bf-f77a-4e9a-81be-47a538637733)

I could add the LAN bridge to another VM and have it pull an IP address from openwrt


What could possible be added to the script

1) Have it create a new basic network bridge for the LAN without any VLAN tag
2) Allow the user to choose yes or no to create the VLAN tag so an existing linux bridge can be used.

This is a very hand setup and I have use pfsense in the same manner but want to see how well openwrt would perform.

**Reply by sgtjamez** [reply]
I was able to get the VM up and running the way thomasbra1 suggested, but I still have the same issue that led me here... once I start the VPN, I lose the GUI. Is this a firewall issue? I followed the setup routine outlined on Novaspirit Tech's [video](https://www.youtube.com/watch?v=3mPbrunpjpk&t=622s).

Please someone help me!
